src/css/styles/_c-lang.scss

```diff
@@ -0,0 +1,63 @@
+.c-lang {
+  position: absolute;
+  top: 1rem;
+  right: 0;
+
+  &__toggle {
+    cursor: pointer;
+  }
+
+  &__toggle-text {
+    padding: 1rem 0.5rem 0.5rem !important;
+  }
+
+  &__icon,
+  &__icon-hide {
+    max-width: 12px;
+  }
+
+  &__dd {
+    position: absolute;
+    top: 1.7rem;
+    padding: 0;
+    list-style: none;
+    background: transparent;
+  }
+
+  &-dd__option {
+    position: relative;
+    padding: 0.3rem 1rem 0.3rem 0;
+    text-align: center;
+
+    &::before {
+      position: absolute;
+      top: 0.1rem;
+      left: 0;
+      width: 3rem;
+      content: "";
+      border-top: 1px solid $col-nav-link;
+    }
+
+    &__link {
+      padding: 0.5rem 1.5rem 0.5rem 0.5rem !important;
+      font-family: $font-praktika-bold;
+    }
+  }
+
+  &--mobile {
+    padding: 1rem 0;
+
+    &__item {
```

Comment:
here we would have `.c-lang--mobile__item` instead of `.c-lang__item--mobile`

- [ ] element before modifer